README.md

```diff
@@ -1,2 +1,47 @@
 # docker-browserstack-local
 Docker official Image packaging for Browserstack Local Binary
+
+### Installation
+- Clone the repository
+- Run ``` docker build  -t <build name:tag> <path to cloned folder>``` 
+
+### Running the binary inside container with default command
+ - Run the docker container using  
+- ```docker run -it <build name:tag> --key <browserstack local testing access key>```
+- The default entrypoint runs the binary without any arguments.
+
+### Running the binary inside container with custom arguments
+In order to run binary with custom [arguments](https://browserstack.com/local-testing) one can simply run the binary as follows.
+`docker run <build name:tag> <arguments>`
+
+Eg . Running binary inside container with force local and verbose 3 enabled
+`docker run -it browserstack/local --key=<access key> --verbose 3 --force-local`
+
+### Running the binary with a default custom set command
+- Edit the Dockerfile and replace the content in ENTRYPOINT with whatever command or script you wish to execute. 
+- Optionally you can also set the `BS_KEY` env variable inside the image itself. 
+- After editing save the Dockerfile and build the docker image using the following command.
+- Building the docker image
+- `docker build -t <build: tagname> <path to dockerfile>`
+- Now the image should be compatible to run the custom default command specified.
+
+### Running tests for a server running on my network. 
```

Comment:
this section is only for localhost requests right? make the title more specific?